app/routes/orders/active_items.js

```diff
@@ -1,3 +1,8 @@
 import detail from "./detail";
 
-export default detail.extend({});
+export default detail.extend({
+  async setupController(controller, model = {}) {
+    await this._super(controller, model);
+    controller.on();
```

Comment:
If you're calling `on(), I'm guessing you should probably call `off()` when leaving the page :)